.github/scripts/repair-phase-containment-regions.ps1

```
@@ -0,0 +1,598 @@
+            $entries.Insert(0, ($leading -join "`n"))
+        }
+        return , $entries.ToArray()
+    }
+
+    # No sequence items: the region is a single mapping.
+    $single = ($lines | Where-Object { -not [string]::IsNullOrWhiteSpace($_) }) -join "`n"
+    if ([string]::IsNullOrWhiteSpace($single)) { return , @() }
+    return , @($single)
+}
+
+function Set-EntryFindingKey {
```

PSScriptAnalyzer (1.25.0)

[warning] Function 'Set-EntryFindingKey' has verb that could change system state. Therefore, the function has to support 'ShouldProcess'.

(PSUseShouldProcessForStateChangingFunctions)